samples/testapp/src/commonMain/kotlin/com/android/identity/testapp/TestAppUtils.kt

```diff
@@ -521,7 +521,8 @@ object TestAppUtils {
                     issuer = Issuer(
                         "https://example-issuer.com",
                         dsKey.publicKey.curve.defaultSigningAlgorithmFullySpecified,
-                        "key-1" // TODO: why pass key-id?
+                        "key-1", // TODO: why pass key-id?
+                        X509CertChain(listOf(dsCert))
```

Comment:
I wonder why we're passing `"key-1"` here? I think I left that TODO a while ago... I think I now understand that key-id is only useful to find the DS key out of band? If so, we should probably just pass `null` here instead?